### PR TITLE
Fix typo and the content

### PR DESCRIPTION
**Pull request by howardpen9**

### CODE REVIEW

File: pages/_meta.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
   "index": {
     "title": "Getting started",
-    "display": "hidden",
-    "type": "page"
+    "type": "page",
+    "display": "hidden"
   },
   "start": {
-    "title": "Getting started",
+    "title": "Getting Start",
```
Comment (anton-trunov) [inline pages/_meta.json:8]: "getting started" is correct

File: pages/index.mdx [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 ### Content
 
-- [Getting Started](/start)
```
Comment (anton-trunov) [inline pages/index.mdx:11]: same here

File: pages/language/guides/functions.mdx [hunk 3]
```diff
@@ -58,8 +63,9 @@ extends mutates fun pow(self: Int, c: Int) {
 
 Native functions are direct binding to a func functions:
 
-> **Note**
-> Native functions could be also mutable and extension one
+<Callout type="success" emoji="ℹ️">
+  Native functions could be also mutable and extension one
```
Comment (anton-trunov) [inline pages/language/guides/functions.mdx:67]:
```suggestion
  Native functions could be also mutable and extension ones
```

File: pages/start/index.mdx [hunk 4]
```diff
@@ -1,23 +1,29 @@
 # Welcome to Tact
 
-There are two ways to start with Tact: using a template or starting from scratch.
+**There are two ways to start with Tact**: using a template or starting from scratch.
```
Comment (anton-trunov) [inline pages/start/index.mdx:3]: let's not overuse bold fonts

File: pages/start/index.mdx [hunk 5]
```diff
@@ -1,23 +1,29 @@
...
 
 We recommend using a template, as it contains a **simple contract** that can be deployed to the TON blockchain, along with examples of implementing unit tests and helper functions for contract deployment.
 
-## Getting started from template
+---
+
+## Getting start from template
```
Comment (anton-trunov) [inline pages/start/index.mdx:9]: there should not be any change here

File: pages/start/index.mdx [hunk 6]
```diff
@@ -1,23 +1,29 @@
...
 
 To get started, you can use the template project. It contains a simple contract that can be deployed to the TON blockchain, as well as examples of implementing unit tests and helper functions for contract deployment.
 
 > To create a project from the template, simply create a new repository from the template project: https://github.com/tact-lang/tact-template.
 
-## Getting started from scratch
+## Getting start from scratch
```
Comment (anton-trunov) [inline pages/start/index.mdx:15]: same

File: pages/start/index.mdx [hunk 7]
```diff
@@ -1,23 +1,29 @@
...
 
 Tact is distributed via `npm` package manager and is meant to be installed to TypeScript/JavaScript projects:
 
 ```bash
 yarn add @tact-lang/compiler
 ```
 
+> More details can find in https://github.com/tact-lang/tact
```
Comment (anton-trunov) [inline pages/start/index.mdx:23]:
```suggestion
> More details can be found at https://github.com/tact-lang/tact
```

**Review by anton-trunov — CHANGES_REQUESTED** [review]

File: pages/start/index.mdx (hunk 6 above)
Comment (anton-trunov) [reply pages/start/index.mdx:15]: Getting started is actually grammatically correct

**Comment by tactfunc** [comment]
<3

**Comment by anton-trunov** [comment]
Since there are some merge conflicts and unresolved conversations, I'm inclined to close this PR for now. Feel free to reopen it if you have some time to work on it

**Pull request closed without merging**